pantheon/src/test/java/tech/pegasys/pantheon/cli/CommandTestAbstract.java

```diff
@@ -79,6 +83,13 @@
   @Captor ArgumentCaptor<PermissioningConfiguration> permissioningConfigurationArgumentCaptor;
   @Captor ArgumentCaptor<Collection<URI>> uriListArgumentCaptor;
 
+  @Rule public final TemporaryFolder temp = new TemporaryFolder();
+
+  @Before
+  public void resetSystemProps() {
+    System.setProperty("pantheon.docker", "false");
```

Comment:
This appears to be setting a system property but never sets it back in an `@After`.